tests/system/alerts/test_alerts.py

```diff
@@ -46,6 +46,7 @@ class TestAlerts(TestMLRunSystem):
     # Set image to "<repo>/mlrun:<tag>" for local testing
     image: typing.Optional[str] = None
 
+    @pytest.mark.skip(reason="TODO Test disabled temporarily")
```

Comment:
Instead of skipping it, please just remove the smoke mark (or comment it with TODO)